VMBackup/main/Utils/EventLoggerUtil.py

```diff
@@ -268,6 +268,9 @@ def _dispose(self, disposing):
                             logger.log("Warning: Unable to process events before termination of extension. Exception: {0}" .format(str(ex)))
                 self.disposed = True
                 print("Information: Event Logger has terminated")
+                print("Clearing the temp directory")
+                if os.path.exists(self.temporary_directory):
+                    os.rmdir(self.temporary_directory)
```

Comment:
I think this can be moved to a helper method with a try-catch to suppress failure?